docs/sdks/client-sdks/flutter/quickstart.mdx

```diff
@@ -0,0 +1,290 @@
+- **Default Value**: Fallback value if assignment fails or rules don't match
+
+```dart
+// Make an assignment
+final String variation = Eppo.getStringAssignment(
+  'my-neat-feature',
+  'default-value',
+);
+```
+
+### Assignment Types
+
+The SDK provides different assignment functions based on the type of value you need:
+
+| Function                 | Return Type            |
+| ------------------------ | ---------------------- |
+| `getBooleanAssignment()` | `bool`                 |
+| `getIntegerAssignment()` | `int`                  |
+| `getNumericAssignment()` | `num`                  |
+| `getStringAssignment()`  | `String`               |
+| `getJSONAssignment()`    | `Map<String, dynamic>` |
+
+### Using Assignments
+
+After receiving an assignment, your application should implement logic to modify the user experience accordingly:
+
+```dart
+final String variation = Eppo.getStringAssignment(
+  'my-neat-feature',
+  'default-value',
+);
+
+// Render different components based on assignment
+switch(variation) {
+  case 'landing-page-a':
+    return renderLandingPageA();
+  case 'landing-page-b':
+    return renderLandingPageB();
+  default:
+    return renderLandingPageC();
+}
+```
+
+## Experiments
+
+While feature flags are useful, they do not send you any information about how your users are interacting with the feature. 
+Experiments provide a way to collect data about these interactions using whichever logging and data warehousing system you prefer.
+
+To log events through the SDK, you need to provide a logger when initializing the client:
+
+```dart
+/// Represents an assignment event.
+class AssignmentEvent {
+  /// The allocation the subject was assigned to
+  final String? allocation;
+
+  /// The experiment identifier
+  final String? experiment;
+
+  /// The feature flag identifier
+  final String featureFlag;
+
+  /// The format of the assignment
+  final String format;
+
+  /// The variation key that was assigned
+  final String? variation;
+
+  /// The subject identifier
+  final String subject;
+
+  /// The timestamp of the assignment
+  final String timestamp;
+
+  /// The subject attributes
+  final Map<String, dynamic>? subjectAttributes;
+
+  /// Additional metadata
+  final Map<String, dynamic>? metaData;
+}
+
+// Implement your own `AssignmentLogger`
+class YourAssignmentLogger implements AssignmentLogger {
+  @override
+  void logAssignment(AssignmentEvent event) {
+    // Your logging logic here
+  }
+}
+
+await Eppo.initialize(
+  PrecomputeArguments(subject: subject),
+  SdkOptions(
+    sdkKey: sdkKey,
+    assignmentLogger: YourAssignmentLogger(),
+  ),
+);
+```
+
+:::note
+For your production application, we have documentation on how to set up logging with multiple popular data warehouses and 
+logging systems in the [Assignment page](/sdks/client-sdks/flutter/assignments/#logging-data-to-your-data-warehouse).
+:::
+
+## Contextual Bandits
+
+For bandits, a separate logger handles its logging:
```

Comment:
Maybe also mention how actions need to provided as well. Love how the code example has this here!